packages/iris-grid/src/Formatter.test.ts

```diff
@@ -9,30 +9,38 @@ import {
 } from './formatters';
 import TableUtils from './TableUtils';
 
-function makeFormatter(...settings) {
+function makeFormatter(...settings: ConstructorParameters<typeof Formatter>) {
   return new Formatter(...settings);
 }
 
-const TYPE_DATETIME = 'io.deephaven.time.DateTime';
+const TYPE_DATETIME = TableUtils.dataType.DATETIME;
```

Comment:
This can stay actually and just 1 spot where it's used needs to be changed to `TableUtils.dataType.DATETIME`. That's when creating the formatting rule because that method should use the normalized type from what I can tell. `makeColumnFormattingRule` is only called by `IrisGrid.jsx` with the normalized type, so I restricted that function to expect the normalized type.

I'll revert this string change and update the column creation in this test. Getting the formatting rule can use any type and `getColumnFormat` normalizes the type.

Maybe `makeColumnFormattingRule` should take a string and normalize the type itself, but I wasn't looking to change any functionality in this change.